plan/plan.txt

```diff
@@ -0,0 +1,17 @@
+Overview: The site will be a blog for news relating to the Lego company which will pertain to various events and
+new products being released. It won't have many pages, consisting of the main page, an articles page, an 
+about us page, and a link to the lego website. It would be more static, with it's responsiveness mainly 
+pertaining to opening articles and managing them with types of news and a search bar.
+Goals: The main goals of the site would be to help inform people who enjoy lego about the things the lego 
```

Comment:
I was hoping that you would have more than one goal for this site, along with objectives for each.